src/components/layout/Navigator.tsx

```diff
@@ -0,0 +1,114 @@
+import { useTranslation } from 'react-i18next';
+import { useLocation, useMatch } from 'react-router-dom';
+
+import { ItemRecord } from '@graasp/sdk/frontend';
+import { HomeMenu, ItemMenu, Navigation } from '@graasp/ui';
+
+import { NAVIGATOR_BACKGROUND_COLOR } from '../../config/constants';
+import {
+  HOME_PATH,
+  SHARED_ITEMS_PATH,
+  buildItemPath,
+} from '../../config/paths';
+import { hooks } from '../../config/queryClient';
+import {
+  BREADCRUMBS_NAVIGATOR_ID,
+  HOME_MENU_DROPDOWN_BUTTON_ID,
+  HOME_MENU_ID,
+  buildBreadcrumbsItemLink,
+  buildMenuItemId,
+  buildNavigationDropDownId,
+} from '../../config/selectors';
+
+const {
+  useItem,
+  useParents,
+  useCurrentMember,
+  useChildren,
+  useOwnItems,
+  useSharedItems,
+} = hooks;
+
+const Navigator = (): JSX.Element => {
+  const { t } = useTranslation();
+  const match = useMatch(buildItemPath());
+  const { pathname } = useLocation();
+  const itemId = match?.params?.itemId;
+  const { data: currentMember } = useCurrentMember();
+  const { data: item, isLoading: isItemLoading } = useItem(itemId);
+  const itemPath = item?.path;
+
+  const { data: parents, isLoading: areParentsLoading } = useParents({
+    id: itemId,
+    path: itemPath,
+    enabled: !!itemPath,
+  });
+
+  const isParentOwned =
+    (item?.creator?.id ?? parents?.first()?.creator?.id) === currentMember?.id;
+
+  if (isItemLoading || areParentsLoading) {
+    return null;
+  }
+
+  const buildToItemPath = (id) => buildItemPath(id);
+
+  const menu = [
+    { name: t('Home'), id: 'home', to: HOME_PATH },
+    { name: t('Shared Items'), id: 'shared', to: SHARED_ITEMS_PATH },
```

Comment:
Are there no strings in translations for analytics ?

Reply:
We could use the strings in common since these are common to all 4 front-ends.

Reply:
I'll create a PR to add graasp-translations in general.